packages/studio-ui/package.json

```diff
@@ -23,6 +23,7 @@
     "@analytics/segment": "^1.1.3",
     "@blueprintjs/core": "^3.23.1",
     "@blueprintjs/select": "^3.12.0",
+    "@botpress/superinput": "workspace:^",
```

Comment:
Why the caret?

Reply:
I just ran 'yarn workspace @botpress/studio-ui add @botpress/superinput' I'm just it's just to keep installing the latest build or something